Descriptions: ld should not allow SHT_PROGBITS .eh_frame sections on amd64
 See http://web.archiveorange.com/archive/v/35IZCJfgcLgR4x7TKych#rq048e3BWPe7VNf,
 http://mail.opensolaris.org/pipermail/onnv-notify/2010-March/011731.html
diff -dubr Python-2.7.3/Modules/_ctypes/libffi/src/x86/unix64.S Python-2.7.3.xxx/Modules/_ctypes/libffi/src/x86/unix64.S
--- Python-2.7.3/Modules/_ctypes/libffi/src/x86/unix64.S	2012-04-10 03:07:33.000000000 +0400
+++ Python-2.7.3.xxx/Modules/_ctypes/libffi/src/x86/unix64.S	2012-09-18 11:57:15.700539011 +0400
@@ -324,7 +324,7 @@
 .LUW9:
 	.size	ffi_closure_unix64,.-ffi_closure_unix64
 
-	.section	.eh_frame,"a",@progbits
+	.section	.eh_frame,"a",@unwind
 .Lframe1:
 	.long	.LECIE1-.LSCIE1		/* CIE Length */
 .LSCIE1:
